src/atoms/Button/Button.web.js

```diff
@@ -1,20 +1,351 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
+import styled, { ThemeContext } from 'styled-components';
 
-const Button = ({ type, onClick, children, ...rest }) => (
-  <button onClick={onClick} type={type} {...rest}>
-    {children}
-  </button>
-);
+import Flex from '../Flex';
+import Size from '../Size';
+import Space from '../Space';
+import Icon from '../Icon';
+import Text from '../Text';
+import View from '../View';
+import automation from '../../_helpers/automation-attributes';
+import { getColor, makePxValue, getVariantColorKeys } from '../../_helpers/theme';
+
+const styles = {
```

Comment:
There are also `focus` and `hover` states for Button on the web. Refer Figma.

Reply:
Added!